cdisc_rules_engine/dataset_builders/base_dataset_builder.py

```diff
@@ -54,19 +55,22 @@ def build(self) -> pd.DataFrame:
         pass
 
     def get_dataset(self, **kwargs):
-        # If validating dataset content, ensure split datasets are handled.
-        if is_split_dataset(self.datasets, self.domain):
-            # Handle split datasets for content checks.
-            # A content check is any check that is not in the list of rule types
-            dataset: pd.DataFrame = self.data_service.concat_split_datasets(
-                func_to_call=self.build,
-                dataset_names=self.get_corresponding_datasets_names(),
-                **kwargs,
-            )
-        else:
-            # single dataset. the most common case
-            dataset: pd.DataFrame = self.build()
-        return dataset
+        try:
```

Comment:
This block should not perform any schema validation. Schema validation should be handled in the define dataset builders

Reply:
This should not be changed. We should especially not be converting all exceptions to FailedSchemaValidation execeptions